pep-0484.txt

```diff
@@ -2047,6 +2047,10 @@ Convenience definitions:
 * @no_type_check_decorator, a decorator to create your own decorators
   with the same meaning as ``@no_type_check`` (see below)
 
+* @type_check_only, a decorator only available during type checking
```

Comment:
```suggestion
* ``@type_check_only``, a decorator only available during type checking
```

Reply:
While this makes sense to me, the other definitions in this list are also not marked for some reason.